src/autopas/utils/SimilarityFunctions.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file SimilarityFunctions.h
+ * @author J. Kroll
+ * @date 01.06.2020
+ */
+
+#pragma once
+
+#include "ThreeDimensionalMapping.h"
+#include "autopas/containers/ParticleContainerInterface.h"
+#include "autopas/molecularDynamics/MoleculeLJ.h"
+
+namespace autopas::utils {
+
+/**
+ * calculates homogeneity and max density of given AutoPas simulation
+ * homogeneity > 0.0, normally < 1.0, but for extreme scenarios > 1.0
+ * maxDensity > 0.0, normally < 3.0, but for extreme scenarios >> 3.0
+ *
+ * @tparam Particle
+ * @param container container of current simulation
+ * @return {homogeneity, maxDensity}
```

Comment:
I guess the reason for combining the two is to avoid iterating everything twice? Mention that in the documentation. Also since you write in full sentences add `.` at the end of sentences and use capitalization ;)

Reply:
resolved by commit 5d64489f8fe659f14ef7b5bbc52c69f6219fb882